### DISCUSSION TITLE
Can i update my current git clone of funkin without having to git clone again?

### DISCUSSION THREAD

**Discussion by Stompyyyy** [FunkinCrew/Funkin#3675] [Compiling help] [answered]
So the first step to compiling is to git clone right? Can i update (for example) the 0.5.0 git cloned folder into the 0.5.2 one without doing the git clone again?

**Comment by NotHyper-474** [comment, marked as answer]
~~Do a `git fetch` then a `git pull`~~

~~Or maybe just a `pull`, I forgot how git works~~ Just a `git pull` is enough

Then do `git submodule update` to make sure it updates the assets as well

**Reply by Stompyyyy** [reply]
alr i will look more into it!

**Reply by Stompyyyy** [reply]
That seemed to do it!
You just have to CD into the funkin folder and do a `git pull --all`
